### ISSUE TITLE
Can we stop upgrading modules and adding badges but really writing some code?

### ISSUE THREAD

**Issue by fraserxu** [react-china/forum-frontend#29]
It's a good thing that we're trying to keep update to the latest cool stuff(#28), but seems that we're taking to much time on it.

There are already enough best practise here but I think it's time to actually write some really code. Like 

* sort out the router
* hook the api with discourse
* create real actions or stores

It's not a good sign that we already have [43 commits](https://github.com/react-china/forum-frontend/commits/master) but none of them are real logic code related to this project.

I'd suggest we stop sending unrelated pull request but actually write some code and get shit done.

cc/ @react-china/forum-frontend

**Comment by hysios** [comment]
yes, but sorry, I very busy in this moments, so please push something code, i'd merge
